src/server/middleware/sendHtml.js

```diff
@@ -57,7 +57,19 @@ export function safeSend(res, ...payload) {
   }
 }
 
-export function renderStaticErrorPage(res) {
+export async function fetchCustomErrorPage(fallbackUrl, res) {
+  const response = await fetch(fallbackUrl);
```

Comment:
This `fetch` has 6 sec timeout right?

Reply:
That is a good question, it is the poly filled fetch from `cross-fetch`. Where would I see if there is a timeout?

Reply:
I would add a 3 sec timeout https://github.com/americanexpress/fetch-enhancers#createtimeoutfetch-server--browser for quality.

Reply:
It rejects if it exceeds timeout. I think `fetch` needs to be `createTimeoutFetch` from the enhancers.

Reply:
This is still needed @JamesSingleton using `createTimeoutFetch`